benchmarks/dashboard-stack/README.md

```diff
@@ -0,0 +1,23 @@
+# S3 Benchmark CDK kit
+
+This will deploy an EC2 instance with the S3Canary on it, and will dump a run to CloudWatch
```

Comment:
Seems to be talking about the benchmarks stack but it's in the dashboard stack folder.

Reply:
I'll probably delete it, it's out of date.